src/pages/taehyunKim/Login/Login.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from "react";
+import { withRouter } from "react-router-dom";
+import "./Login.scss";
+import "../../../Styles/reset.scss";
+import "../../../Styles/common.scss";
+
+class LoginTaehyun extends Component {
+  constructor() {
+    super();
+    this.state = {
+      id: "",
+      password: "",
+      isActiveBtn: false,
+    };
+  }
+
+  handleIdInput = (e) => {
+    this.setState({ id: e.target.value });
+  };
+
+  handlePwInput = (e) => {
+    this.setState({ password: e.target.value });
+  };
+
+  checkLogin = () => {
+    this.state.id.indexOf("@") && this.state.password.length >= 5 ? this.setState({ isActiveBtn: true }) : this.setState({ isActiveBtn: false });
+  };
```

Comment:
```suggestion
  handleIdInput = (e) => {
    this.setState({ id: e.target.value }, () => {
      this.checkLogin();
    });
  };

  handlePwInput = (e) => {
    this.setState({ password: e.target.value }, () => {
      this.checkLogin();
    });
  };
```
아마 setState가 비동기 연산이기 때문에 바로 효과를 보려고 onKeyUp에 이벤트를 다신 것 같은데 
이렇게 하시면, setState가 완료된 시점에 checkLogin()메소드를 부를 수 있게 되어 정상적으로 작동하실 것 같습니다.

동일한 문법으로
```suggestion
  handleIdInput = async (e) => {
    await this.setState({ id: e.target.value });
    this.checkLogin();
  };

  handlePwInput = async (e) => {
    await this.setState({ password: e.target.value });
    this.checkLogin();
  };
```

이런 식으로 명시적으로 동기적 행동을 유발시킬 수 있습니다

Reply:
그렇군요, 공부를 좀 더 철저히 해야겠습니다 감사합니다👍